.github/workflows/docs-check.yml

```diff
@@ -36,6 +36,24 @@ jobs:
         with:
           python-version: 3.9
 
+      # use markdownlint to check for lint issues
+      - uses: DavidAnson/markdownlint-cli2-action@v16
+        if: always()
+        with:
+          globs: 'Documentation/**/*.md,!Documentation/Helm-Charts'
+          config: .markdownlint.yaml
+
+      # markdown lint still allows multi-paragraph list items to be aligned to fewer tab-spaces than
+      # will properly render with mkdocs. use 'prettier' to enforce all markdown should be aligned
+      # to a 4-line boundary
+      - name: Prettify code
+        if: always()
+        uses: creyD/prettier_action@v4.3
+        with:
+          dry: true
+          # --no-error-on-unmatched-pattern -- don't error on symlinks
+          prettier_options: --no-error-on-unmatched-pattern --check **/*.md
```

Comment:
if the CI get fail what cmd we should run locally to update the docs?
And can we document it or error out.

Reply:
I added the vscode settings for prettier, as well as a suggestion for rook users to install the tool. It'll autoformat markdown when saving files.